DocSum/ui/gradio/docsum_ui_gradio.py

```diff
@@ -201,6 +234,22 @@ def render(self):
                     )
             submit_btn.click(fn=self.generate_summary, inputs=[input_text], outputs=[generated_text])
 
+        with gr.Blocks() as url_ui:
+            # URL text UI
+            with gr.Row():
+                with gr.Column():
+                    input_text = gr.TextArea(
+                        label="Please paste a URL for summarization",
+                        placeholder="Paste a URL for the information you need to summarize",
+                    )
+                    submit_btn = gr.Button("Generate Summary")
+                with gr.Column():
+                    generated_text = gr.TextArea(
+                        label="Text Summary", placeholder="Summarized text will be displayed here"
+                    )
+            submit_btn.click(lambda input_text: self.generate_summary(self.read_url(input_text)), inputs=input_text, outputs=generated_text)
+            # submit_btn.click(fn=self.generate_summary, inputs=[input_text], outputs=[generated_text])
```

Comment:
remove this line --> # submit_btn.click(fn=self.generate_summary, inputs=[input_text], outputs=[generated_text])